browser-test/src/applicant/application_navigation.test.ts

```diff
@@ -1515,8 +1521,11 @@ test.describe('Applicant navigation flow', () => {
 
         await expect(addressCorrectionInput).toHaveValue('true')
 
-        // Set thing to soft eligibilty
-        await adminPrograms.toggleEligibilityGating()
+        // Set program to soft eligibility
```

Comment:
nitpick: I don't think you need this comment anymore since it's clear it's being set to to not gating

Reply:
Updated!